jetty9/src/main/docker/Dockerfile

```diff
@@ -39,8 +39,9 @@ RUN mkdir webapps $TMPDIR \
  && chown -R jetty:jetty $JETTY_BASE $TMPDIR
 
 # Start Jetty directly
-COPY app-env.bash /
-RUN chmod +rx /app-env.bash
+COPY setup-env-ext.bash /tmp/
+RUN cat /tmp/setup-env-ext.bash >> /setup-env.bash \
```

Comment:
The `#/bin/bash` is not a problem as it is still a valid comment.   However it is unnecessary and we should instead replace it with just a comment saying that the file is expected to be appended to `setup-env.bash`